content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Connect your database'
+metaTitle: 'Connect your database'
+metaDescription: 'Connect your database to your project'
+langSwitcher: ['typescript', 'node']
+dbSwitcher: ['mongodb']
+toc: false
+---
+
+## Connect your database
+
+To connect your database, you need to set the `url` field of the `datasource` block in your Prisma schema to your database [connection URL](../../../../../reference/database-reference/connection-urls):
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "mongodb"
+  url      = env("DATABASE_URL")
+}
+```
+
+In this case, the `url` is [set via an environment variable](../../../../concepts/more/environment-variables) which is defined in `.env`:
+
+```js file=.env
+DATABASE_URL ='mongodb+srv://test:test@cluster0.ns1yp.mongodb.net/myFirstDatabase'
+```
+
+Because MongoDB is currently a preview feature, you need to explicitly define that in the [`generator`]() block.
+
+```prisma file=prisma/schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["mongodb"]
```

Comment:
Correct spelling of the preview feature @matthewmueller?

Reply:
Changed to mongoDb everywhere - same as formatter